### PR TITLE
Fix condition reason constants

### PR DESCRIPTION
**Pull request by luksa**
Add "Reason" suffix to some consts & remove message constants that aren't used anywhere

### CODE REVIEW

**Comment by jboyd01** [comment]
@luksa having issues with gofmt on pkg/controller/controller_instance.go

File: pkg/controller/controller_instance.go [hunk 1]
```diff
@@ -53,37 +53,31 @@ const (
...
 	errorErrorCallingProvisionReason           string = "ErrorCallingProvision"
 	errorUpdateInstanceCallFailedReason        string = "UpdateInstanceCallFailed"
 	errorErrorCallingUpdateInstanceReason      string = "ErrorCallingUpdateInstance"
 	errorDeprovisionCallFailedReason           string = "DeprovisionCallFailed"
 	errorDeprovisionBlockedByCredentialsReason string = "DeprovisionBlockedByExistingCredentials"
 	errorPollingLastOperationReason            string = "ErrorPollingLastOperation"
-	errorWithOriginatingIdentity               string = "Error with Originating Identity"
-	errorWithOngoingAsyncOperation             string = "ErrorAsyncOperationInProgress"
-	errorWithOngoingAsyncOperationMessage      string = "Another operation for this service instance is in progress. "
+	errorWithOriginatingIdentityReason         string = "Error with Originating Identity"
```
[the remark this answers is not in the document]
Comment (luksa) [reply pkg/controller/controller_instance.go]: I'm not changing the string in this PR. I'm just changing the name of the constant. 

But I agree this needs to be fixed - in a separate PR. But can we even change this or would such a change represent an API change?

[the remark this answers is not in the document]
Comment (luksa) [reply pkg/controller/controller_instance.go]: I've filed an issue: https://github.com/kubernetes-incubator/service-catalog/issues/2530

**Comment by jboyd01** [comment]
@luksa you have a merge conflict that needs resolving

**Pull request merged**